python_scripts/02_basic_preprocessing.py

```diff
@@ -363,9 +339,10 @@ def plot_cv_indices(cv, X, y, ax, lw=20):
 
     # Formatting
     yticklabels = list(range(n_splits))
-    ax.set(yticks=np.arange(n_splits+2) + .5, yticklabels=yticklabels,
-           xlabel='Sample index', ylabel="CV iteration",
-           ylim=[n_splits + .2, -.2], xlim=[0, 100])
+    ax.set(yticks=np.arange(n_splits + 2) + .5,
+           yticklabels=yticklabels, xlabel='Sample index',
+           ylabel="CV iteration", ylim=[n_splits + .2,
+                                        -.2], xlim=[0, 100])
```

Comment:
not very clean, but we can always workaround by setting `ylim =  [n_splits + .2, -.2]` on a new line and then pass `ylim=ylim`.